### PR TITLE
Adding Rados Tier-2 test cases

### PR DESCRIPTION
**Pull request by pdhiran**
Additions: Email Alerts configuration

Signed-off-by: Pawan Dhiran <pdhiran@redhat.com>

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs

# Checklist:

- [x] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [x] Implement the test script and perform test runs
- [x] Submit PR for code review and approve
- [x] Update Polarin Test with Automation script details and update automation fields
- [x] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test

### CODE REVIEW

**Comment by pdhiran** [comment]
logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-F1N9SH/

File: conf/pacific/rados/tier_2_rados.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          - node-exporter
+          - crash
+        no-of-volumes: 4
+        disk-size: 20
+      node11:
+        role:
+          - mon
+          - rgw
+          - node-exporter
+          - crash
```
Comment (psathyan) [inline conf/pacific/rados/tier_2_rados.yaml:78]: It is a good practice to end with a newline character

File: suites/pacific/rados/tier_2_rados.yaml [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      config:
+        email_alerts:
+          smtp_host: smtp.corp.redhat.com
+          smtp_sender: ceph-iad2-c01-lab.mgr@redhat.com
+          smtp_port: 25
+          interval: 10
+          smtp_destination:
+            - pdhiran@redhat.com
+          smtp_from_name: Rados Sanity Cluster Alerts
+      desc: Configure email alerts on ceph cluster
```
Comment (psathyan) [inline suites/pacific/rados/tier_2_rados.yaml:113]: It is a good practice to with a newline character

**Review by psathyan — COMMENTED** [review]
Looks good to me... minor comments

File: suites/pacific/rados/tier_2_rados.yaml (hunk 2 above)
Comment (pdhiran) [reply suites/pacific/rados/tier_2_rados.yaml]: made changes

**Review by psathyan — APPROVED** [review]

**Review by tintumathew10 — APPROVED** [review]

**Pull request merged**